scripts/launch-prover-service

```diff
@@ -1,13 +1,19 @@
 #!/bin/bash
-set -e
+# If ESPRESSO_SEQUENCER_LIGHTCLIENT_ADDRESS is unset and
+# ESPRESSO_SEQUENCER_DEPLOY_LIGHTCLIENT_CONTRACT is "yes" or "1" the light
+# client contract will be deployed. This feature is intended for demo or testnet
+# purposes only.
+set -eo pipefail
 
 if [[ -z "${ESPRESSO_SEQUENCER_LIGHTCLIENT_ADDRESS}" ]]; then
-  if [[ "${ESPRESSO_SEQUENCER_DEPLOY_LIGHTCLIENT_CONTRACT}" ]]; then
+  if [[ "${ESPRESSO_SEQUENCER_DEPLOY_LIGHTCLIENT_CONTRACT}" == "yes" || "${ESPRESSO_SEQUENCER_DEPLOY_LIGHTCLIENT_CONTRACT}" == "1" ]]; then
     echo "Light client contract address not found, deploying it now."
     ./gen-demo-contract --contracts-root-dir="./contracts"
     forge fmt
     forge build
-    ESPRESSO_SEQUENCER_LIGHTCLIENT_ADDRESS=$(forge script contracts/script/LightClientTest.s.sol --fork-url $ESPRESSO_SEQUENCER_L1_PROVIDER --broadcast 2>/dev/null | grep "Contract Address:" | tail -1 | sed "s/^.*: //")
+    forge script DeployLightClientDemoScript --fork-url $ESPRESSO_SEQUENCER_L1_PROVIDER --broadcast
+    ESPRESSO_SEQUENCER_LIGHTCLIENT_ADDRESS=$(cat contracts/broadcast/LightClient.s.sol/*/run-latest.json | jq -r '.receipts[1].contractAddress')
```

Comment:
Should we also change the contract script file name to `LightClient.demo.s.sol`?